ex2_hal/athena/equipment_handler/source/deployablecontrol.c

```diff
@@ -109,9 +109,3 @@ bool switchstatus(Deployable_t sw) {
         return 1; // check this
     }
 }
-
-// This should return 1, assuming switches are not connected to ground when undepressed
-bool deploy(Deployable_t deployable) {
```

Comment:
line 157 in general.c calls this, so you probably just get an implicitly defined func warning. You'll have to change that call in general.c to use the new deploy() function.

We'll have -Werror on in no time for things like this I'm sure ;)

Reply:
https://github.com/AlbertaSat/ex2_obc_software/pull/239/commits/ccdfbce9c25baf6844c569abe3cb1ce6d80fa451